src/components/molecules/ChatMessageBox/ChatMessageBox.tsx

```diff
@@ -16,12 +17,14 @@ export interface ChatMessageBoxProps {
   selectedThread?: WithId<MessageToDisplay>;
   sendMessage: SendMesssage;
   sendThreadReply: SendChatReply;
+  isQuestion: boolean;
```

Comment:
In fact, I wonder if it would be better to just pass `activeMode` through here, maybe called `selectedChatMode` / `currentChatMode` / `chatMode` or similar.

Ideally this would just use `ChatModeType` (which is an `enum`, as suggested elsewhere in this review) rather than `ChatMode` (which is a nested data object).

That way our API can support any mode we need in future without needing code changes.